crates/yttrium/src/chain_abstraction/client.rs

```diff
@@ -131,3 +362,32 @@ impl Client {
         }
     }
 }
+
+#[derive(Debug)]
+pub struct RouteUiFields {
+    pub route: Vec<TxnDetails>,
+    // pub bridge: TxnDetails,
+    pub initial: TxnDetails,
+    pub local_total: Amount,
```

Comment:
what's local_total? fee for all txs? init + route?

Reply:
It's the total fee amount in the user's local currency (e.g. USD)